src/run.rs

```diff
@@ -193,13 +203,18 @@ fn list_image_records(state: &State) -> io::Result<HashMap<String, ImageRecord>>
         }
 
         let image_parts = trimmed_line.split('\t').collect::<Vec<_>>();
-        if let [id, repository, tag, date_str] = image_parts[..] {
+        if let [sha256_id, repository, tag, date_str, size] = image_parts[..] {
             let repository_tag = RepositoryTag {
                 repository: repository.to_owned(),
                 tag: tag.to_owned(),
             };
 
-            match image_records.entry(id.to_owned()) {
+            // The id is in the format:
+            // sha256:745895703263072416be27b333d19eff4494b287001f6c6adddd22b963a3429d
+            // We only want the first 12 characters of the hash.
+            let id = sha256_id.get(7..19).unwrap();
```

Comment:
Repository convention: all `unwrap`s should be justified with a comment (even if obvious).